test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml

```diff
@@ -16,6 +17,7 @@ nemesis_selector: ['limited']
 nemesis_seed: '026'
 nemesis_interval: 30
 nemesis_during_prepare: false
+nemesis_add_node_cnt: 3
```

Comment:
I think it's safe to already make it the default.

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/10345

Reply:
thx, this can be removed from here now.

Reply:
Removed, thanks for the reminder